docs/source/reference/volumes.rst

```diff
@@ -24,7 +24,9 @@ Supported volume types:
 Volumes on Kubernetes
 ---------------------
 
-SkyPilot supports creating and managing PVC (Persistent Volume Claim) volumes on Kubernetes clusters through three commands:
+In Kubernetes clusters, PVCs (Persistent Volume Claims) request and bind to PV (Persistent Volume) resources. These persistent volumes can be backed by various storage backends, including **block storage** solutions (AWS EBS, GCP Persistent Disk) and **distributed file systems** (JuiceFS, Nebius shared file system, AWS EFS, GCP Filestore), etc.
```

Comment:
Was more thinking adding to L18, or a subitem under L18, to say exactly the example backends we have tested. We can say: "Tested: x, y, z". This way, readers looking at the top immediately notice it.

Fine to keep this line.

Reply:
Updated